python/cudf/dataframe/buffer.py

```diff
@@ -62,31 +61,7 @@ def serialize(self, serialize, context=None):
         (header, frames)
             See custom serialization documentation in dask.distributed.
         """
-
-        from cudf.comm.serialize import should_use_ipc
-
-        # Use destination info to determine if we should do IPC.
-        use_ipc = should_use_ipc(context)
-        header = {}
-        # Should use IPC transfer
-        if use_ipc:
-            # Reuse IPC handle from previous call?
-            if self._cached_ipch is not None:
-                ipch = self._cached_ipch
-            else:
-                # Get new IPC handle
-                ipch = rmm.get_ipc_handle(self.to_gpu_array())
-
-            header['kind'] = 'ipc'
-            header['mem'], frames = serialize(ipch)
-            # Keep IPC handle alive
-            self._cached_ipch = ipch
-        # Not using IPC transfer
-        else:
-            header['kind'] = 'normal'
-            # Serialize the buffer as a numpy array
-            header['mem'], frames = serialize(self.to_array())
-        return header, frames
+        return {}, [self.to_gpu_array()]
```

Comment:
This will return by copy instead of by reference in the future.
```suggestion
        return {}, [self.mem]
```